src/app/core/utils/api-token/api-token.service.ts

```diff
@@ -48,8 +55,12 @@ export class ApiTokenService {
     private cookiesService: CookiesService,
     private router: Router,
     private store: Store,
+    oAuthService: OAuthService,
+    oAuthConfigurationService: OAuthConfigurationService,
     appRef: ApplicationRef
   ) {
+    oAuthConfigurationService.config$.subscribe(config => oAuthService.configure(config));
+
     const initialCookie = this.parseCookie();
```

Comment:
I think so. The oauth library should only handle the management of the api-token. Nevertheless the PWA needs to store some other informations (is user anonymous, has user a basket or an order) to restore informations on reload. This could be done with the initial cookie handling. It can be discussed, whether the apiToken should be part of the cookie or not.